src/states/Lobby.gd

```diff
@@ -70,7 +72,6 @@ func player_connected(player_id: int, info: Dictionary) -> void:
 func player_disconnected(player_id: int) -> void:
 	update_table()
 	if get_tree().is_network_server():
-		# warning-ignore:return_value_discarded
```

Comment:
Did we change this in editor config?

Reply:
Yep we did that recently
https://github.com/kdreese/godot-3D-shooter-project/commit/2257cc6c2dfe1633efc258495af44f532100cac5